definitions/types/cidr.json

```diff
@@ -1,6 +1,10 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
   "type": "string",
+  "pattern": "^([0-9]{1,3}\\.){3}[0-9]{1,3}(\/([0-9]|[1-2][0-9]|3[0-2]))?$",
```

Comment:
this might need to support mask only cidrs (like kubernetes) `/16`

probably worth having a fullCidr / maskOnlyCidr types

Reply:
I think this is fine since this type is called a CIDR and we don't want to conflate a CIDR and a mask.

Here is a simple regex that will match **either** a cidr or a mask:
`^(?:(?:[0-9]{1,3}\.){3}[0-9]{1,3})?(?:\/(?:[0-9]|[1-2][0-9]|3[0-2]))?$`

We can just use that in the handful of places that GCP supports either cidrs or masks @WillBeebe (GKE and maybe subnets)?

Reply:
@chrisghill @WillBeebe  I added a `kube-cidr` type in [d7dde26](https://github.com/massdriver-cloud/massdriver-bundles/commit/d7dde26c9ee600bef02fdb188b97d0f175fc8b4d) with a regex to match a proper cidr range or just subnet mask and opened this draft to use it. LMK if that seems right.

Reply:
@jaketf since this "cidr-or-mask" type currently only exists in 2 fields of the `gcp-gke-cluster` bundle, for now I think its best to just input the pattern in the `gcp-gke-cluster` hardcoded.

If we see this pattern showing up in other bundles or situations, I think we can make a `cidr-or-mask.json` type, since its not really specific to kubernetes (if anything its a GCP-ism).

Let's get the CIDR changes merged and I'll make comments on that GKE PR 👍